### PR TITLE
fix: pass hammer instans to picassoHammer

### PR DESCRIPTION
**Pull request by haxxmaxx**
picassoHammer needs an instance for hammer passed to it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,11 @@
...
   "dependencies": {
     "css-loader": "^2.1.0",
+    "hammerjs": "2.0.8",
     "html-loader": "^0.5.5",
-    "picasso-plugin-hammer": "^0.25.3",
-    "picasso-plugin-q": "^0.25.3",
-    "picasso.js": "^0.25.3",
+    "picasso-plugin-hammer": "2.2.7",
+    "picasso-plugin-q": "2.2.7",
+    "picasso.js": "2.2.7",
```
Comment (haxxmaxx) [inline package.json:36]: Didn't work with the current version of picasso. Bumped to the latest, but might be risky moving 2 major versions at once?

Comment (haxxmaxx) [reply package.json:36]: Also, should we pin them? i can do that in a different PR first for all the deps if we want that

Comment (niekvanstaveren) [reply package.json:36]: As long as we test it properly it should be fine to bump major versions

Comment (haxxmaxx) [reply package.json:36]: manually tested. download image/pdf is broken, but it was before as well.
I've also pinned the rest of the deps, to the version that is in package-lock

**Review by niekvanstaveren — APPROVED** [review]
Looks okay to me

**Pull request merged**